### ISSUE TITLE
Account Signup Issue - Unused phone numbers report "This phone number has already been used"

### ISSUE THREAD

**Issue by TimCliff** [steemit/condenser#1483]
There have been cases where new users try to sign up, and it says their phone number is in use. I confirmed with one of the users and they are 100% sure they have not tried to create an account before. The details of the account will be provided to @roadscape over a private channel.

**Comment by pfunks** [comment]
From just now, in steemit.chat #help channel 

![image](https://user-images.githubusercontent.com/22420130/27210812-a69fcc56-521a-11e7-8ae2-2dc08bc2d804.png)

**Comment by roadscape** [comment]
Still an issue?

**Comment by TimCliff** [comment]
I've had a few reports within the last week. I don't know if they started the process before all the latest changes were applied though. I will track and report back with details if I encounter any more.

**Comment by TimCliff** [comment]
Three more users reported the issue today. I'll try to get more details and message you.

**Comment by sneak** [comment]
@valzav please fix this ASAP

**Comment by valzav** [comment]
I'm pretty sure that the phones were really used to create accounts already, we need to check this on phone by phone basis..

**Comment by TimCliff** [comment]
I have been screening users who have reported it, and as far as I can tell they seem to be legitimate first time signups. 

I provided roadscape and andrarchy 3 phone numbers over private chat, and a fourth user emailed their number to sneak.

**Comment by TimCliff** [comment]
@valzav - I sent you the details as well.

**Comment by Priksh** [comment]
Hi,
 I am trying to create an account on steemit but it tells me i have used my number already, that is not possible. how do i sign up?

**Comment by TimCliff** [comment]
Priksh - please ask for help in the #help channel of Steemit chat.

**Comment by Wallazz** [comment]
same issue, what can i do?

**Comment by TimCliff** [comment]
@Wallazz - message me on steemit chat and I will try to help.

**Comment by osnabrugge** [comment]
I have the same issue. I tried to create an account, but it keeps telling me my phone number is in use. How can I message you on steemit chat if I don't have an account?  Kind of a catch 22.

**Comment by TimCliff** [comment]
@osnabrugge steemit chat does not require a Steem blockchain account. You can create an account there even if you are having trouble getting your Steem blockchain account approved via the steemit.com signup process.

**Comment by osnabrugge** [comment]
I've tried registering on steemit chat as well. No welcome email. Checked my junk folder. Perhaps that may have been the issue for the block chain account as well?  I remember getting a text when I tried signing up there, but never received any emails.

**Comment by TimCliff** [comment]
GitHub is really not an appropriate place for this conversation. Can you try discord chat: https://discord.gg/HYj4yvw?

**Comment by osnabrugge** [comment]
Agreed. I just had no way of getting help, hence posting on GitHub. I'm in discord now and posted in #help.

**Comment by Wallazz** [comment]
" I've tried registering on steemit chat as well. No welcome email. Checked
my junk folder. Perhaps that may have been the issue for the block chain
account as well? I remember getting a text when I tried signing up there,
but never received any emails. " same issue here

2018-02-18 21:21 GMT-08:00 Sean <notifications@github.com>:

**Comment by Zekt** [comment]
Just to mention that currently a phone number become "been used" right after someone use it in registration, but not yet use the confirmation code, so anyone could just type in a random number then the number get "been used" and the owner can no longer use it.

**Comment by TimCliff** [comment]
@Zekt that is not how it works. It only counts the number as used after the SMS code is successfully verified.

**Comment by evsinoz** [comment]
Same problem. I started the registration process and got the number already in use. Tried a 2nd time (different user name and email address) but same. Any ideas on how to contact Steemit or how to get this respolved? Thanks

**Comment by Zekt** [comment]
@TimCliff I'm pretty sure that's what happened to me, I took the sign up process with a random username and my phone number, received a code and never use it, now I can't finish the process with another username anymore, it always say my number is used.

**Comment by evsinoz** [comment]
Thanks for the update.

Regards
Phil


On 1 Mar 2018, at 14:57, Viktor Lin <notifications@github.com<mailto:notifications@github.com>> wrote:


@TimCliff<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ftimcliff&data=02%7C01%7C%7C976bf166639c4ef7371008d57f30e207%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636554770281688018&sdata=z6hekE0KXXMVT7lM5ZxuGWP9qm3NYR0eV5nuJGYIDT8%3D&reserved=0> I'm pretty sure that's what happened to me, I took the sign up process with a random username and my phone number, received a code and never use it, now I can't finish the process with another username anymore, it always say my number is used.

**Comment by TimCliff** [comment]
GitHub is not the appropriate place for signup issues. Please go to steemit chat #help if you need help.

**Comment by asteriospapastamatakis** [comment]
here the same... i put my name (first time trying to register) said my phone is already un use... then i tried to do it from the beginning and i put the username i choose in the beginning and now it says ''username reserved'' because iv passed the mail verification .. any clue of what should i do?

**Comment by TimCliff** [comment]
@asteriospapastamatakis ask for help in the help channel of steem chat.

**Comment by asteriospapastamatakis** [comment]
thank you very much
